types/pouchdb-core/index.d.ts

```diff
@@ -55,6 +55,15 @@ declare namespace PouchDB {
             rev: RevisionId;
             status: Availability;
         }
+        interface RevisionDiffList {
+            [documentId : string]: string[];
+        }
+        interface RevisionDiff {
+            [change : string]: string[];
```

Comment:
My understanding is that this corresponds to [this example response](https://pouchdb.com/api.html#example-response-20) from the docs, and that the key tells you which *kind* of change you're dealing with.

@nolanlawson Is there a fixed set of keys here (e.g. "missing")?  If so we can use an enum / union type.